root/account/applications/RevokeApplicationAccess.js

```diff
@@ -8,18 +8,32 @@
  */
 
 import ConfirmLayout from '../../components/ConfirmLayout.js';
+import {ACCESS_SCOPE_PERMISSIONS} from '../../constants.js';
+import {commaOnlyListText}
+  from '../../static/scripts/common/i18n/commaOnlyList.js';
 
 type Props = {
+  +application: ApplicationT,
   +form: SecureConfirmFormT,
+  +permissions: $ReadOnlyArray<number>,
 };
 
 const RevokeApplicationAccess = ({
+  application,
   form,
+  permissions,
 }: Props): React$Element<typeof ConfirmLayout> => (
   <ConfirmLayout
     form={form}
-    question={l(
-      `Are you sure you want to revoke this application's access?`,
+    question={texp.l(
+      `You’re about to revoke {app}’s permissions to: {permissions}.
+       Are you sure you want to revoke this application's access?`,
+      {
+        app: application.name,
+        permissions: commaOnlyListText(permissions.map(permission => (
+          ACCESS_SCOPE_PERMISSIONS[permission]()
```

Comment:
Permissions can be pretty long to read. It would probably easier to have these as an itemized list since this page is nearly empty and can afford some more lines, or at the very least in italic (**untested** suggestion).

```suggestion
        permissions: <i>{commaOnlyListText(permissions.map(permission => (
          ACCESS_SCOPE_PERMISSIONS[permission]()}</i>
```

Reply:
That's a very good point, and I didn't think about the fact that `question` supports proper React anyway, so I amended it, this seems a lot better to me:

![image_2023-07-24_225730694](https://github.com/metabrainz/musicbrainz-server/assets/1069224/bddc0aeb-52fa-48ff-b145-eae87eeec1ce)

Reply:
“ListenBrainz Prod” looks like a label name :smile: